src/goose/console.clj

```diff
@@ -1,60 +1,63 @@
 (ns goose.console
+    "Functions to serve the console web interface"
   (:require [clojure.string :as str]
             [goose.broker :as b]
+    [goose.defaults :as d]
             [hiccup.page :refer [html5 include-css include-js]]
             [ring.middleware.keyword-params :as ring-keyword-params]
             [ring.middleware.params :as ring-params]
             [ring.util.response :as response]))
 
 ;; Page handlers
-(defn load-css
-  "Load static css file"
+(defn ^:no-doc load-css
+      "Load goose console's static css file"
   [_]
-  (-> "css/style.css"
+      (-> "css/style.css"
       response/resource-response
       (response/header "Content-Type" "text/css")))
 
-(defn load-img
+(defn ^:no-doc load-img
   "Load goose logo"
   [_]
   (-> "img/goose-logo.png"
       response/resource-response
       (response/header "Content-Type" "image/png")))
 
-(defn load-js
-  "Load javascript file"
+(defn ^:no-doc load-js
+      "Load goose console's javascript file"
   [_]
   (-> "js/index.js"
       response/resource-response
       (response/header "Content-Type" "text/javascript")))
 
-(defn redirect-to-home-page
-  "Redirects the user to the home page using given prefix-route function"
+(defn ^:no-doc redirect-to-home-page
+      "Redirects the user to the homepage"
   [{:keys [prefix-route]}]
   (response/redirect (prefix-route "/")))
 
-(defn not-found
-  "A default not found response"
+(defn ^:no-doc not-found
+      "Default not found response"
   [_]
   (response/not-found "<div> Not found </div>"))
 
 ;; View components
-(defn layout
+(defn ^:no-doc layout
   "Generates HTML layout for webpage using provided hiccup components
 
-  ### Args
+   ### Args
 
-  components: A variadic list of functions. Each function is expected to accept
-              a map `data` and return hiccup data representing part of a webpage
+   components: A variadic list of functions. Each function is expected to accept
+   a map `data` and return hiccup data representing part of a webpage
 
-  ### Usage
-   ```Clojure
-   (def job [job]
-        [:p {:class \"job\"} (:id job)])
+   ### Usage
+   ```clojure
+    (defn job [j]
+      [:div (:id j)])
```

Comment:
question(non-blocking): I don't understand this change, can you explain what it is?

Reply:
This is an improved example for what a component accepted by layout function looks like.
If you notice, the previous definition was wrong, and I added a `div` instead of `:p`.